web_widget_x2many_2d_matrix/README.rst

```diff
@@ -50,15 +53,25 @@ field_label_y_axis
     Use another field to display in the table header
 field_value
     Show this field as value
+field_editability
+    Use this field to evaluate if a cell should be editable or not. It
```

Comment:
imagine I can return

```python
{
    'disabled': 'disabled', # that's what you need
    'class': 'some class', # we can add some fancyness per field
    'onwhatever': 'injected javascript', # might be helpful in some use cases
    'style': 'display: none', # or just hide it?
    'out of ideas': 'but theres probably more cool stuff to do with this',
}
```

and all keys become attributes of the input. Very flexible.